spf.go

```diff
@@ -60,10 +60,10 @@ type IPMatcherFunc func(ip net.IP, name string) (bool, error)
 // Resolver provides abstraction for DNS layer
 type Resolver interface {
```

Comment:
The new output parameter TTL is not documented on the interface methods

Reply:
done